### PR TITLE
HRQB 7 - Initial app scaffolding

### PR DESCRIPTION
**Pull request by ghukill**
### Purpose and background context

This PR establishes some initial repository scaffolding, including:
- rename app to `hrqb`
- provide instructions for developing in `arm64` architecture (e.g. Apple Silicon chips)
- Oracle and luigi dependencies
- establish `unit` and `integration` pytest markers to utilize if/where needed going forward

### How can a reviewer manually see the effects of these changes?

Not much to see at this point, just preparing the repository for future work.

Except, if time affords, appreciate any feedback on the `docs/arm64_installation.md` instructions.  Following these would allow for local development on Apple Silicon chips (e.g. M1, M2, etc.) using an Oracle client built for `x86` architecture.  Like Carbon, development by using the Docker container is an option as well, but I personally found it helpful to take this time to set this up, then have a pure python environment to work with.  

### Includes new or updated dependencies?
YES

### Changes expectations for external applications?
NO

### What are the relevant tickets?
- https://mitlibraries.atlassian.net/browse/HRQB-7

### Developer
- [X] All new ENV is documented in README
- [X] All new ENV has been added to staging and production environments
- [X] All related Jira tickets are linked in commit message(s)
- [X] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer(s)
- [ ] The commit message is clear and follows our guidelines (not just this PR message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The provided documentation is sufficient for understanding any new functionality introduced
- [ ] Any manual tests have been performed **or** provided examples verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: docs/arm64_installation.md [hunk 1]
```diff
@@ -1,3 +1,184 @@
...
+```
+
+### 2- Install an `x86` version of Homebrew
+
+This will install an additional `x86` version of Homebrew alongside any pre-existing `arm64` version installed
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Add the following to your `~/.zprofile` to ensure the correct version of homebrew is used for new shell
```
Comment (ehanson8) [inline docs/arm64_installation.md:46]: "Run `nano ~/.zprofile` (or other preferred editor) to ensure..." to be more explicit?

**Review by ehanson8 — APPROVED** [review]
Installation instructions worked after some collaborative troubleshooting, one suggestion for clarity in addition to the recommendations discussed on Slack

File: docs/arm64_installation.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+library files that were installed
+```shell
+DYLD_LIBRARY_PATH=/usr/local/lib:$DYLD_LIBRARY_PATH
+```
+
+### 8- Confirm virtual environment works as expected
+
+With the `x86` virtual environment is created, we can confirm it's working as expected.
+
+First, completely close out the current shell and open a new one in this app project directory.  This
```
Comment (jonavellecuerdo) [inline docs/arm64_installation.md:157]: [Non-blocking] I want to make sure I understand how this works. In Step 6, we created a Python virtual environment that uses the x86 Python that we installed _through_ the `x86`/`i386` architecture environment via Rosetta, right?

When we close out of the current x86 shell, how is it that the arm64 terminal can run the Oracle client library?

**Review by jonavellecuerdo — APPROVED** [review]
@ghukill Thanks for putting this together! I agree that it is useful to have the "arm64_installation" instructions available for any apps that require working with the MIT Data Warehouse. Just had one non-blocking question for ya'!

File: docs/arm64_installation.md (hunk 2 above)
Comment (ghukill) [reply docs/arm64_installation.md:157]: @jonavellecuerdo - excellent question!  That's the beauty and the maddening thing about all this.  All that effort results in a python virtual environment that is compiled for `i386` architecture.  Then, when running `pipenv shell` or `pipenv run <CMD>` and this python installation is used, it's compiled for `i386` architecture.

I dont' want to misspeak here... I haven't quite fully wrapped my head around why you don't need to invoke an `x86` shell to run the `i386` python virtual environment, but in my testing it just doesn't seem to be the case.

I'd like to push on that a bit more for a more thorough understanding.

**Pull request merged**